src/xpcc/architecture/driver/atomic/queue.hpp

```diff
@@ -67,6 +67,9 @@ namespace xpcc
 			ALWAYS_INLINE bool
 			isFull() const;
 			
+			ALWAYS_INLINE bool
+			isNotFull() const { return not isFull(); }
```

Comment:
> I'm always amazed at some of the push back I get for this though

I didn't mean to offend you in any way, I was just genuinely curious about the use of these operators.

> Now excuse me, while I climb onto my high horse. Bwahaha.

Like [this](https://pocketsergeant.files.wordpress.com/2016/02/wp-1455927201540.jpg) one? :D